Fix `error: ignoring return value of function declared with 'warn_unused_result' attribute [-Werror=unused-result]`.
Unlike Clang, GCC does not allow suppressing the warning with a plain (void) cast.
---

diff -uNr a/src/tclsqlite.c b/src/tclsqlite.c
--- a/src/tclsqlite.c
+++ b/src/tclsqlite.c
@@ -4562,7 +4562,7 @@
       fprintf(stderr,
           "attach debugger to process %d and press any key to continue.\n",
           GETPID());
-      fgetc(stdin);
+      (void)!fgetc(stdin);
     }else{
 #if defined(_WIN32) || defined(WIN32)
       DebugBreak();
